packages/evolution-common/src/services/widgets/WidgetConfig.ts

```diff
@@ -37,6 +37,7 @@ export type InputStringType<CustomSurvey, CustomHousehold, CustomHome, CustomPer
     maxLength?: number;
     datatype?: 'string' | 'integer' | 'float';
     size?: 'large' | 'small' | 'medium';
+    textTransform?: string;
```

Comment:
There might be a better type than string for this, like 'none' | 'capitalize' | 'uppercase' | 'lowercase'. I think that can be sufficient for us.

Reply:
Fixed in commit 87c2e27 🙂